Miami task

novak-k opened the pull request:
- [DEMO LINK](https://novak-k.github.io/layout_miami/)

valerii-derkach reviewed (changes requested):
remove displaying this error
<img width="516" alt="image" src="https://user-images.githubusercontent.com/102413088/182180737-e52c8736-6818-49dc-a6cd-b08b962e2a82.png">


make the address open in a new tab 
<img width="277" alt="image" src="https://user-images.githubusercontent.com/102413088/182180945-76cf0e73-3f15-47cf-a83a-707b7aa6acdf.png">
add some smooth hover to links

too much scale for pictures, make them do not overcover others 
<img width="1224" alt="image" src="https://user-images.githubusercontent.com/102413088/182181329-6169f1af-617a-4fbe-84ae-8d43cf09bc18.png">

Anna-Tkachenko reviewed (changes requested):
Good job!
Some comments:
1) When I try submit an empty form - I don't see any errors, as user, I need to see that all fields are required
2) I can't submit the form. When I submit the form with filled fields - I don't see any changes. Form should be cleared after submit
3) Logo in footer redirects to 404 error page - fix it

novak-k commented:
I have problems with that submit form working with the e.preventDefault() method. So I have changed "click" on "submit". And it's helped. But now the page is reloading as I understand but shouldn't. 

With the logo in footer it was just a typo, fixed.

alexandra-protyanova commented:
<img width="517" alt="Снимок экрана 2022-08-08 в 15 20 43" src="https://user-images.githubusercontent.com/102182808/183416838-644a72c8-9256-41d9-a649-1df95aeae4e8.png">
On submit of form i was redirected to 404

alexandra-protyanova reviewed (changes requested):
Fix comments

novak-k commented:
It was fixed, as far as I understand. <form action="#">

alexandra-protyanova reviewed (changes requested):
On form-submit page shouldn't be reload

novak-k commented:
> 

done, main js fixed

alexandra-protyanova reviewed (approved):
🔥
